pyproject.toml

```diff
@@ -6,12 +6,6 @@ build-backend = "setuptools.build_meta"
 # Equivalent to use_scm_version=True
 [tool.setuptools_scm]
 
-# Tool Config
-[tool.isort]
-profile = "black"
-src_paths = ["jira", "tests"]
```

Comment:
is this not needed anymore (for black .... I didnt study yet on the pyproject.toml config stuff)

Reply:
not needed as `ruff` implements `isort` so we have a `tool.ruff.isort` section